compute/cloud-client/src/test/java/compute/disks/HyperdisksIT.java

```diff
@@ -96,15 +96,15 @@ public void stage1_CreateHyperdiskTest()
     Assert.assertTrue(hyperdisk.getZone().contains(ZONE));
   }
 
-  @Disabled
   @Test
   public void stage1_CreateHyperdiskStoragePoolTest()
       throws IOException, ExecutionException, InterruptedException, TimeoutException {
     String poolType = String.format("projects/%s/zones/%s/storagePoolTypes/hyperdisk-balanced",
```

Comment:
this value mismatches the value and instructions in the code sample. please align between the two. either the code sample should enforce this format or the test should use the same as the code sample

Reply:
It's not a scope of this task. I've created a new task to fix code and will follow yours recommendations.

Reply:
The third test uses link to the StoragePool created in the second test.

Reply:
I think you misunderstood the comment.  I've added the comment and code change suggestion to `CreateHyperdiskStoragePool.java`.

Reply:
As I can see, the path to the storage pool is identical to the path in CreateHyperdiskStoragePool in lines 39-40. Please correct me if I'm wrong.